Basic/APIDiorama/Assets/Common/Scripts/NetworkListExample.cs

```diff
@@ -0,0 +1,52 @@
+using Unity.Collections;
+using Unity.Netcode;
+using UnityEngine;
+
+public class NetworkListExample : NetworkBehaviour
```

Comment:
was this meant to be commited? This shouldn't be in /common, comments should be cleaned up and comments adding description on the lesson should be added

Reply:
I wanted to make a sample scee out of it at the beginning, but ended up never doing so. We can remove it.
@SamuelBellomo are there any other blockers?

Reply:
Removed